Sources/Rownd/Models/Context/Auth.swift

```diff
@@ -90,10 +90,14 @@ func authReducer(action: Action, state: AuthState?) -> AuthState {
 // MARK: Token / auth API calls
 
 struct TokenRequest: Codable {
-    var refreshToken: String
+    var refreshToken: String?
+    var id_token: String?
```

Comment:
Use swift convention here `idToken` and `appId` and then use the `CodingKeys` to translate to the proper string format.

Reply:
updated